verticapy/connection/connect.py

```diff
@@ -125,6 +129,52 @@ def connect(section: str, dsn: Optional[str] = None) -> None:
         prev_conn.close()
     try:
         gb_conn.set_connection(vertica_connection(section, dsn), section, dsn)
+        print("Connected Successfuly!")
+    except OAuthTokenRefreshError as e:
+        error_message = str(e)
+
+        from verticapy.connection.write import new_connection
+
+        # Check if the error message contains the refresh token part
+        if "Failed getting OAuth access token from a refresh token" in error_message:
+            # Handle the specific error related to the Refresh token part
+
+            print("Refresh Token or Client ID is not valid. Please re-enter")
```

Comment:
same here

Reply:
@oualib  Shouldn't we display the errors always? This is just summarizing an error.